### ISSUE TITLE
FileCompiler should ignore the word include inside comments

### ISSUE THREAD

**Issue by Rayden** [ryancramerdesign/ProcessWire#1752]
Here are two examples of code that i generaly use, in which the file compiler modifies the content inside the comments:

```
<?php

/**
 * Include functions and globals
 *
 */
include_once("./_func.php");
include_once("./_init.php");
```

and

```
<?php

/**
 * Main HTML include
 *
 */

echo  "<!doctype html>".
      "<html class='no-js' lang='nl' itemscope itemtype='http://schema.org/Article'>".
      "<head>".
      "<meta charset='utf-8'>".
      "<meta http-equiv='X-UA-Compatible' content='IE=edge'>".
      "<title>{$ph->page_title}</title>".
      "<meta name='description' content='{$ph->meta_description}'>".
      "<meta name='viewport' content='width=device-width, initial-scale=1'>";
```

As soon it sees the word "include" it will modify it. This causes parse errors in the code.

```
<?php

/**
 * Functions  include(\ProcessWire\wire('files')->compile(*/Users/raymond/Dropbox/www_domains/projectname/htdocs/site/templates/,array('includes'=>true,'namespace'=>true,'modules'=>true)))
 */
```

and

```
<?php

/**
 * Main HTML  include(\ProcessWire\wire('files')->compile(*/Users/raymond/Dropbox/www_domains/projectname/htdocs/site/templates/,array('includes'=>true,'namespace'=>true,'modules'=>true)))
 */

echo  "<!doctype html>".
      "<html class='no-js' lang='nl' itemscope itemtype='http://schema.org/Article'>".
      "<head>".
      "<meta charset='utf-8'>".
      "<meta http-equiv='X-UA-Compatible' content='IE=edge'>".
      "<title>{$ph->page_title}</title>".
      "<meta name='description' content='{$ph->meta_description}'>".
      "<meta name='viewport' content='width=device-width, initial-scale=1'>";
```

Solution: the FileCompiler should ignore the word "include" inside comments

**Comment by Da-Fecto** [comment]
+1

**Comment by ryancramerdesign** [comment]
I'm not seeing FileCompiler attempt to do anything either any of the indicated examples. I'm wondering if this one might have been fixed already? Are you still seeing the compiler error there? Thanks.

**Comment by Rayden** [comment]
I can confirm its fixed now (3.0.13)
